portfolio/src/main/webapp/script.js

```diff
@@ -56,7 +56,19 @@ function openContact(){
 async function serverCommunication(){
```

Comment:
nit: This could probably have a better name that describes what it does. This becomes important when other people have to read and make changes to your code.